components/ringcentral/ringcentral.app.mjs

```diff
@@ -185,6 +185,13 @@ export default {
         ...args,
       });
     },
+    async getMessage(
+      accountId = "~", extensionId, messageId
+    ) {
+      return this.makeRequest({
+        path: `/account/${accountId}/extension/${extensionId}/message-store/${messageId}`,
+      });
+    },
```

Comment:
If we update `getMessage` to accept `...args`, we can pass along the `$` variable to `makeRequest`.
```suggestion
    async getMessage({
      accountId = "~", extensionId, messageId, ...args
    } = {}) {
      return this.makeRequest({
        path: `/account/${accountId}/extension/${extensionId}/message-store/${messageId}`,
        ...args,
      });
    },
```